CHANGELOG.md

```diff
@@ -5,42 +5,43 @@ CHANGELOG
 ------
 
 **ENHANCEMENTS**
-- Add new configuration parameter `Scheduling/SlurmSettings/QueueUpdateStrategy` to allow cluster update when
-  `SlurmQueues` configuration changes don't impact Slurm scheduler configuration.
-- Add support for multiple Elastic File Systems.
-- Add support for multiple FSx File Systems.
-- Add support for attaching existing FSx for Ontap and FSx for OpenZFS File Systems.
-- Add support for FSx Lustre Persistent_2 deployment type.
-- Add support for memory-based scheduling in Slurm.
-  - Configure `RealMemory` on compute nodes by default as 95% of the EC2 memory.
-  - Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based scheduling in Slurm.
+- Add support for memory-based job scheduling in Slurm
+  - Configure compute nodes real memory in the Slurm cluster configuration.
+  - Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to enable memory-based scheduling in Slurm.
   - Add new configuration parameter `Scheduling/SlurmQueues/ComputeResources/SchedulableMemory` to override default value of the memory seen by the scheduler on compute nodes.
+- Improve flexibility on cluster configuration updates to avoid the stop and start of the entire cluster whenever possible.
+  - Add new configuration parameter `Scheduling/SlurmSettings/QueueUpdateStrategy` to set the preferred strategy to adopt for compute nodes needing a configuration update and replacement.
+- Add support to mount existing FSx for ONTAP and FSx for OpenZFS file systems.
+- Add support to mount multiple instances of existing EFS, FSx for Lustre / for ONTAP/ for OpenZFS file systems.
+- Add support for FSx Lustre Persistent_2 deployment type.
 - Prompt user to enable EFA for supported instance types when using `pcluster configure` wizard.
-- Change default EBS volume types from gp2 to gp3 in both the root and additional volumes.
 - Add support for rebooting compute nodes via Slurm.
 
 **CHANGES**
-- Remove support for Python 3.6.
 - Upgrade Slurm to version 21.08.8-2.
-- Do not require `PlacementGroup/Enabled` to be set to `true` when passing an existing `PlacementGroup/Id`.
+- Upgrade EFA installer to version 1.17.2 
+  - ---TBC---
```

Comment:
We can copy/paste from here https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/CHANGELOG.md

Reply:
Done, together with the rest of software update that was missing...